src/main/java/io/eigr/spawn/api/exceptions/SpawnException.java

```diff
@@ -0,0 +1,25 @@
+package io.eigr.spawn.api.exceptions;
+
+/**
+ * Generic Spawn exception.
+ *
+ * @author Paulo H3nrique Alves
+ */
+public class SpawnException extends RuntimeException {
```

Comment:
I would not like to use unhandleable exceptions. I think we can extend IllegalStateException
```suggestion
public class SpawnException extends IllegalStateException {
```

Reply:
I believe for do that it's necessary extends Exception directly, it will force to handle  errors.